### PR TITLE
[DOCS] EQL: Clarify until keyword docs

### PR DESCRIPTION
**Pull request by jrodewig**
Changes:
* Revises the `until` keyword content for clarity
* Simplifies and replaces the `until` example 
* Updates the examples in the PID tip to follow ECS

### Preview
https://elasticsearch_61794.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/eql-syntax.html#eql-until-keyword

### CODE REVIEW

File: docs/reference/eql/syntax.asciidoc [hunk 1]
```diff
@@ -514,41 +514,48 @@ sequence by user.name with maxspan=15m
 [[eql-until-keyword]]
 ==== `until` keyword
 
-You can use the `until` keyword to specify an expiration event for sequences.
-Matching sequences must end before this event, which is not included the
-results. If this event occurs within a sequence, the sequence is not considered
-a match.
+You can use the `until` keyword to specify an expiration event for a sequence.
```
Comment (rw-access) [inline docs/reference/eql/syntax.asciidoc:517]: this is great, thank you @jrodewig!

**Review by rw-access — APPROVED** [review]
LGTM!

**Review by dgieselaar — APPROVED** [review]
Thanks! Super helpful changes.

**Comment by jrodewig** [comment]
Thanks @rw-access @dgieselaar!

**Pull request merged**